universal-login-relayer/lib/services/WalletService.ts

```diff
@@ -1,28 +1,43 @@
-import ProxyContract from 'universal-login-contracts/build/Proxy';
-import WalletContract from 'universal-login-contracts/build/Wallet';
-import WalletMasterContract from 'universal-login-contracts/build/WalletMaster';
-import LegacyWallet from 'universal-login-contracts/build/LegacyWallet';
+const WalletContract = require('universal-login-contracts/build/Wallet');
+const WalletMasterContract = require('universal-login-contracts/build/WalletMaster');
+const ProxyContract = require('universal-login-contracts/build/Proxy');
+const LegacyWallet = require('universal-login-contracts/build/LegacyWallet');
 import {hasEnoughToken, isAddKeyCall, getKeyFromData, isAddKeysCall} from '../utils/utils';
-import {utils, Contract, ContractFactory} from 'ethers';
+import {utils, ContractFactory, Wallet, providers} from 'ethers';
 import defaultDeployOptions from '../config/defaultDeployOptions';
+import ENSService from './ensService';
+import AuthorisationService from './authorisationService';
+import {EventEmitter} from 'fbemitter';
 
+declare interface Message {
+  to: string;
+  from: string;
+  value: string;
+  data: string;
+  nonce: number;
+  gasPrice: string;
+  gasToken: string;
+  gasLimit: string;
+  operationType: number;
+  signature: string;
+}
 
 class WalletService {
-  constructor(wallet, walletMasterAddress, ensService, authorisationService, hooks, provider, legacyENS) {
-    this.wallet = wallet;
+  private codec: utils.AbiCoder;
+  private bytecode: string;
+  private abi: utils.Interface;
+  private contractJSON: ContractFactory;
+  private useInitData: boolean;
+
+  constructor(private wallet: Wallet, private walletMasterAddress: string, private ensService: ENSService, private authorisationService: AuthorisationService, private hooks: EventEmitter, private provider: providers.Provider, private legacyENS : boolean) {
```

Comment:
- add a newline in between each parameter
- perhaps add a rule to linter about too long lines?